lib/export/types.go

```diff
@@ -54,7 +54,7 @@ type (
 		TileLocation    string
 
 		SourceTags      map[string]uint `json:",omitempty"`
-		sourceTagsMutex *sync.Mutex
+		SourceTagsMutex *sync.Mutex
```

Comment:
Let's not export this field.

If we need other packages to lock this mutex we should create methods on the struct that do that for us